README.md

```diff
@@ -1,3 +1,122 @@
-# vscode
+# VS Code
 
-The action runs VSCode in web browser.
+This action runs an instance of [VS Code](https://code.visualstudio.com/) and runs in in the default web browser.
+
+It requires the name of the image on which to run VS Code and references to the following 5 volumes: data, code, config, notebooks, and results. 
+These volumes will be mounted to /project/data, /project/modules, /project/config, /project/notebooks and /project/results respectively.
+
+### Quick example
+
+```
+jobs:
+  vscode:
+    action: gh:neuro-actions/vscode@master
```

Comment:
`@v1.0.1` — this is the only repo where we have non `v1.0.0`. I've just added it to make the image `image` parameter optional

Reply:
Fixed in the latest commit